artscii/src/tests/Asciify.test.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from '@jest/globals';
+import convertToGrayScales from '../services/convertToGrayScales';
+import dummyImgSrc from './dummyImgSrc';
+import drawAscii from '../services/drawAscii';
+
+const image = new Image();
+image.src = dummyImgSrc;
+const width = 256;
+const height = 128;
+const canvas = document.createElement('canvas');
+const context = canvas.getContext('2d');
```

Comment:
Unit testing looks great and seems to have good coverage with all testing passed. However, I had to manually `npm install canvas` to get past an exception being thrown here in your test suite. Could you also include the dependency change in the package.json for this PR?

Reply:
Huh, yeah that's strange. I had the same issue. When line 10 gets called (const canvas = document.createElement('canvas');) it just sets canvas to an undefined. Looks like a good suite of tests though!

Reply:
I got the same issue but adding that dependency will fix it. It's a minor one. It looks like a good coverage of unit tests has been implemented.

Reply:
Woops, sorry about that - added the dependency!